packages/plugin-vue/README.md

```diff
@@ -62,6 +62,13 @@ export interface Options {
    * Use custom compiler-sfc instance. Can be used to force a specific version.
    */
   compiler?: typeof _compiler
+
+  /**
+   * Prebundle SFC libs
+   *
+   * @default true
+   */
+  prebundleSfc?: boolean
```

Comment:
```suggestion
  prebundle?: boolean
```

Reply:
resolve in 7a862abcdde5327e781e142cd61959d838466d45